### PR TITLE
chore(security vulnerabilities): npm audit fix

### PR DESCRIPTION
**Pull request by YoDaMa**

### CODE REVIEW

File: build/tools/package.json [hunk 1]
```diff
@@ -18,9 +18,9 @@
   },
   "homepage": "https://github.com/azure/azure-iot-sdk-node#readme",
   "dependencies": {
-    "azure-iothub": "1.10.1",
+    "azure-iothub": "^1.10.1",
```
Comment (pierreca) [inline build/tools/package.json]: >^ [](start = 21, length = 1)

reason?

File: common/transport/amqp/package.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
   "typings": "index.d.ts",
   "dependencies": {
     "async": "^2.6.2",
-    "azure-iot-common": "1.10.1",
+    "azure-iot-common": "^1.10.1",
```
Comment (pierreca) [inline common/transport/amqp/package.json]: >"^ [](start = 24, length = 2)

please keep the strict version declaration for intra-sdk package dependencies

File: package.json [hunk 3]
```diff
@@ -4,7 +4,10 @@
...
 		"mocha": "^5.2.0"
 	},
-	"engines" : {
-		"node" : ">=8.0.0"
+	"engines": {
+		"node": ">=8.0.0"
+	},
+	"dependencies": {
+		"azure-iot-device": "^1.11.0"
 	}
```
Comment (pierreca) [inline package.json]: i don't think this should be here

File: common/transport/amqp/package.json (hunk 2 above)
Comment (YoDaMa) [reply common/transport/amqp/package.json]: I removed all the carets for our packages.

**Pull request merged**